file-service-api/src/main/java/uk/gov/justice/services/fileservice/domain/FileReference.java

```diff
@@ -17,11 +17,17 @@
     private final UUID fileId;
     private final JsonObject metadata;
     private final InputStream contentStream;
+    private final Boolean deleted;
```

Comment:
Do we need this as a Boolean? (with potential null value) or would a primitive suffice.